SETUP_SUBMITTER_CMF.md

```diff
@@ -0,0 +1,120 @@
+# Unreal Submitter/CMF Worker Setup Instructions
+
+This will walk you through setting up your Unreal Submitter with optional additional instructions for setting up an instance to act as a worker as part of a Customer Managed Fleet (CMF). The Unreal Submitter in Deadline Cloud can currently only work if you've set up a CMF and have connected a worker with Unreal installed on an appropriate instance type.
+
+If you only need to install the submitter on an existing machine which already has Unreal installed, you only need to complete the steps "Install Build Tools" through "Submitter Installer" below and can skip the CMF/worker steps.
+
+## Create a Customer Managed Fleet
+
+If you don't yet have a CMF set up, you can complete the instructions below as part of the steps titled "Worker host setup" and "Install software for jobs" as you create the CMF with these instructions: https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-a-cmf.html, and then (If you're using EC2) create an AMI which can function as either your CMF worker or your Submitter.
+
+## Create a new Windows EC2 instance to install Unreal on (Optional)
+
+If you’re setting up on a brand new Windows EC2 Instance either as your submitter or as your CMF worker node, a g5.2xlarge instance with 200 GB of storage will likely be reasonable minimum:
+
+- Download the Epic Installer and install the latest version of Unreal (5.2 or higher is required)
+- NVidia Grid drivers - Follow windows instructions - https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/install-nvidia-driver.html#nvidia-GRID-driver
+
+The following steps: (“Install Build Tools”, "Environment Setup", "Deadline Software Installation", "Build the Plugin") should be followed whether you’re currently setting up your submitter machine or creating an instance/AMI to act as your worker node for Unreal jobs.
+
+## Install Build Tools
+
+The Unreal Submitter Plugin currently must be compiled locally.
+
+- Install Visual Studio 2022 or later using the Visual Studio Installer
```

Comment:
I was thinking it could be helpful to add the link to where to download Visual Studio? https://visualstudio.microsoft.com/

Reply:
Added